lib/solaar/cli/show.py

```diff
@@ -247,11 +247,11 @@ def run(devices, args, find_receiver, find_device):
                 dev_num += 1
         return
 
-    dev = find_receiver(receivers, device_name)
+    dev = find_receiver(devices, device_name)
```

Comment:
Can these changes be amends to previous commits so that the commit history looks better?